### PR TITLE
Change Graph logger call sites

### PR DESCRIPTION
**Pull request by nathaniel-may**
waiting to merge till #4163 is in the feature branch, but it's ready for review.

### Description

Changes log call sites in graph module

TODO:
- [ ] point to feature/structured-logging to merge

### Checklist

- [ ] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [ ] I have run this code in development and it appears to resolve the stated issue
- [ ] This PR includes tests, or tests are not required/relevant for this PR
- [ ] I have updated the `CHANGELOG.md` and added information about my change

### CODE REVIEW

File: core/dbt/events/types.py [hunk 1]
```diff
@@ -264,3 +306,4 @@ def cli_msg(self) -> str:
     SystemStdOutMsg(bmsg=b'')
     SystemStdErrMsg(bmsg=b'')
     SystemReportReturnCode(code=0)
+    SelectorReportInvalidSelector(selector_methods={'':''}, spec_method='', raw_spec='')
```
Comment (emmyoop) [inline core/dbt/events/types.py:309]: missing `SelectorAlertAllUnusedNodes` and `SelectorAlertUpto3UnusedNodes`

File: core/dbt/graph/selector.py [hunk 2]
```diff
@@ -103,11 +94,11 @@ def get_nodes_from_criteria(
         try:
             collected = self.select_included(nodes, spec)
         except InvalidSelectorException:
-            valid_selectors = ", ".join(self.SELECTOR_METHODS)
```
Comment (emmyoop) [inline core/dbt/graph/selector.py:106]: Is there a reason to move this into `SelectorReportInvalidSelector`?

**Review by emmyoop — APPROVED** [review]
Just a small issue.

File: core/dbt/graph/selector.py (hunk 2 above)
Comment (nathaniel-may) [reply core/dbt/graph/selector.py:106]: I'm trying to take the approach where the event represents all the necessary data, and the responsibility to turn data into human-readable strings is entirely up to the event methods. `valid_selectors` looks like it's entirely about turning data into strings so I moved it.

File: core/dbt/events/types.py (hunk 1 above)
Comment (nathaniel-may) [reply core/dbt/events/types.py:309]: good catch!

File: core/dbt/graph/selector.py (hunk 2 above)
Comment (emmyoop) [reply core/dbt/graph/selector.py:106]: Makes sense.  Just want to make sure I'm consistent.

**Pull request merged**